api/.env.example

```diff
@@ -97,10 +97,9 @@ LARATRUST_ENABLE_CACHE=true
 FEATURE_SKILL_LIBRARY=true
 FEATURE_STATUS_NOTIFICATIONS=true
 FEATURE_DIRECTIVE_FORMS=true
-FEATURE_RECORD_OF_DECISION=true
+FEATURE_RECORD_OF_DECISION=false
```

Comment:
Couldn't quite get this resolved with Cypress config, might be due to there being two env files, web and api, that complicates things. The api might not be controllable from Cypress?

Reply:
We have [a spike](https://github.com/GCTC-NTGC/gc-digital-talent/issues/4227) for this so I wouldn't worry about it.

Reply:
Yeah, I tried a suggestion from that issue but couldn't get it to work